ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -157,11 +166,98 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       </div>;
     };
 
+    const removeOldInstitutionalAffiliation = (affiliation: InstitutionalAffiliation) => {
```

Comment:
these 2 are pulled out and made more explicit from their original contexts, which has itself been pulled out to `renderOldInstitutionalAffiliationComponents()`